app/templates/views/templates/attach-pages.html

```diff
@@ -32,10 +33,23 @@ <h1 class="banner-title">{{ error.title }}</h1>
     <p class="govuk-body">
         In total, your letter must be 10 pages or less (5 double-sided sheets of paper).
     </p>
-    <p class="govuk-body">
-        Your file must meet our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_upload_a_letter') }}">letter
-            specification</a>.
-    </p>
+
+{% set letter_spec %}
```

Comment:
Need to update this bit also :)

```suggestion
{% set attachment_spec %}
```